src/main/webapp/js/instructorFeedbacks.js

```diff
@@ -182,35 +182,32 @@ function bindCopyButton() {
         return false;
     });
 }
+var prevRow;
 
 function bindCopyEvents() {
-
-    $('#copyTableModal >tbody>tr').on('click', function(e) {
+    $('#copyTableModal > tbody > tr').on('click', function(e){
         e.preventDefault();
```

Comment:
either this is repeated code, or the 'return false;' at line 214 is repeated.

return false does e.preventDefault() and also does e.stopPropagation(), we should either have e.preventDefault() without the return false or have return false without the e.preventDefault()